tools/scylla-nodetool.cc

```diff
@@ -107,6 +108,23 @@ const auto app_name = "nodetool";
 
 logging::logger nlog(format("scylla-{}", app_name));
 
+std::string doc_link(std::string_view url_tail) {
+    const auto product = scylla_product();
+    const auto version = scylla_version_short();
+
+    const auto prefix = product == "scylla-enterprise" ? "enterprise" : "opensource";
+
+    std::string branch = product == "scylla-enterprise" ? "enterprise" : "master";
+    nlog.info("doc_link(): product={}, version={}", product, version);
+    if (!version.ends_with("~dev")) {
+        std::vector<std::string> components;
+        boost::split(components, version, boost::algorithm::is_any_of("."));
+        branch = fmt::format("branch-{}.{}", components[0], components[1]);
```

Comment:
might want to add an `assert(component.size() >= 2)` here.

Reply:
Yes, good idea, added one in v2.